### ISSUE TITLE
No call backs ?

### ISSUE THREAD

**Issue by nmfarshad** [blockchain/receive-payments-demos#60]
I have developed an app in laravel and call backs are not working ?

When I try to get callback logs its only returning "[]"

Please advice

**Comment by sFaxsy** [comment]
there is no callbacks

**Comment by mitko-janeski** [comment]
I have same problems if we get some support would be great

**Comment by kinklim** [comment]
same problem here. +1

**Comment by ehaaaaaaaaa** [comment]
same problem here. +1

**Comment by hellbeast222** [comment]
Sent from Yahoo Mail on Android 
 
  On Mon, Feb 5, 2018 at 1:28 AM, ehaaaaaaaaa<notifications@github.com> wrote:   
same problem here. +1


—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread. 



LADIES.... MAY I HELP?

**Comment by hellbeast222** [comment]
Sent from Yahoo Mail on Android 
 
  On Mon, Jan 29, 2018 at 11:35 AM, nmfarshad<notifations@github.com> wrote:   
I have developed an app in laravel and call backs are not working ?

When I try to get callback logs its only returning "[]"

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.

EXACTLY WHAT ARE YOU CREATING? A BIT-JACKER? WALLET CRACKER? OR WHAT? I CAN PROBABLY HELP YOU OUT

**Comment by hellbeast222** [comment]
Refer to: blockchain.info/api/api_receive

**Comment by hellbeast222** [comment]
HERE'S YOUR PROBLEM....1) CALLBACKS CANNOT EXCEED 255 CHARACTERS MAX $_GET[`CONFORMATION'] I'll let you know exactly what the problem is, I just need to know is this for double SPENDS?  

if it's something to do multiple
Sent from Yahoo Mail on Android 
 
  On Mon, Jan 29, 2018 at 11:35 AM, nmfarshad<notifications@github.com> wrote:   
I have developed an app in laravel and call backs are not working ?

When I try to get callback logs its only returning "[]"

**Comment by ehaaaaaaaaa** [comment]
Hi, hellbeast222

Here the complete question as below link in stackexchange:
[link](https://bitcoin.stackexchange.com/questions/69992/blockchain-receive-callback-api-in-callback-url-not-working)

Thanks

**Comment by hellbeast222** [comment]
I would suggest testing with a larger payment. The callbacks have been somewhat of an ass pain in my experience. Bc.info receive payments Api WILL NOT WORK FOR THIS.receiving address must be identical also, the funds will remain in their original destination since such dogshit is hierarchical.The wallet balance could be manually refreshed by the user but must have only restricted access as not to exceed api request limits obviously. this is fun. Hold on, I've got an idea. 
Sent from Yahoo Mail on Android 
 
  On Mon, Jan 29, 2018 at 11:35 AM, nmfarshad<notifications@github.com> wrote:   
I have developed an app in laravel and call backs are not working ?

When I try to get callback logs its only returning "[]"

**Comment by hellbeast222** [comment]
Also keep your dirty fucking hands off of the merchandise! Try not leaving your greasy ass fingerprints all over the bitcoin. Any anomalous data throws a red flag immediately. I had the same problem just before I perfected my stupid ass wallet. Another possible solution is to pay me to send you the iron fucking clad, stealth wallet. ////]    Admin@SouthAlabama.US [\\\\####JUST RELAX! and I'll CLEA n ThIS dogshit right UP for yA Chief. ####34esSz7uLQfLVsBY6CHd3mcEAyc6nLtyyr


Sent from Yahoo Mail on Android 
 
  On Mon, Jan 29, 2018 at 11:35 AM, nmfarshad<notifications@github.com> wrote:   
I have developed an app in laravel and call backs are not working ?

When I try to get callback logs its only returning "[]"

**Comment by hellbeast222** [comment]
Ehaaaaaaaaa,
The only problem that I can think of is possibly the amount of your payments are too small and

**Comment by ehaaaaaaaaa** [comment]
Hi, Hellbeast222

I will test it later with large amount.

Thanks your information.

**Comment by mitko-janeski** [comment]
Hellbeast222 I also got same problem tested with multiple payments but still nothing. I suspect might be laravel or who knows its just dumb that no one reply and give correct answer

**Comment by assafY** [comment]
Hi all,

Thank you for reporting this. This was an internal issue that is now fixed.

Please let me know if you experience any more issues.

**Comment by mitko-janeski** [comment]
I still get same problem no change at all, only blank bracket:

[ ]

**Comment by nmfarshad** [comment]
I just did a test with 0.0001359 BTC 
can confirm its working for me !
@assafY Tx Man

**Comment by assafY** [comment]
@mitko-janeski is this for payments received now? Does it happen continuously?

**Comment by mitko-janeski** [comment]
Do you have any live chat or teamviewer so you can see and I can give you more details and examples
https://i.gyazo.com/6a183678e9ff1a0f1e8812d9b3079a0a.png

**Comment by assafY** [comment]
Thanks. There is an issue catching back on callbacks from the past week. We are working on solving this, but in the meantime it would be useful if you can report whether callbacks are sent for new payments or not.

**Comment by mitko-janeski** [comment]
I'm gonna do new payment and let you know. Another question I've wanted to ask is there any minimal payment requirements or can be any amount ?

Once I've tested im gonna report with results.

**Comment by assafY** [comment]
Thanks, much appreciated. Can be any amount.

**Comment by mitko-janeski** [comment]
Ok im getting some information now, but I get:
"responseCode" : 302,
  "rawResponse" : "<!DOCTYPE html>\n<html>\n 

So I guess I need to double check the code for possible errors

Previously I didnt receive any information so was hard to debug, lets hope for the best

**Comment by sFaxsy** [comment]
"calledAt" : 1517856215000,
  "responseCode" : 200,
  "rawResponse" : "Waiting for confirmations"

working fine for me now ,

thanks

**Comment by ehaaaaaaaaa** [comment]
Hi,

Yes, it working fine for me too.

thanks

**Comment by hellbeast222** [comment]
If I'm not mistaken, as of 4 or 5 months ago the I think that the minimum received total was around 5 dollars. This was just around the time of the hard fork and pre segwit. I'm curious to see how this turns out. Also, here is my personal email for sensitive detail and correspondences. I have a few inquiries that may require a bit more privacy. Thanks for the update and the best of luck👍🏻💯

Sent from Yahoo Mail on Android 
 
  On Mon, Feb 5, 2018 at 11:37 AM, Mitko J.<notifications@github.com> wrote:   
I'm gonna do new payment and let you know. Another question I've wanted to ask is there any minimal payment requirements or can be any amount ?

Once I've tested im gonna report with results.

**Comment by hellbeast222** [comment]
💎👌🏻👍🏻💯Congrats Y'all.. Excuse me for following along without much to contribute toward the project, but my computer is fried garbage right now after I was recently the latest victim of the most evil and elusive viruses known to man.  Ironically, I took  part in it's development. I would be especially careful of any API. Especially bc💯👍🏻congrats again and thanks! 💯👌🏻👍🏻Wish 8 could participate and help y help Y'all with this project 

Sent from Yahoo Mail on Android 
 
  On Tue, Feb 6, 2018 at 12:26 AM, Luke McCullough<lukemccullough@yahoo.com> wrote:   If I'm not mistaken, as of 4 or 5 months ago the I think that the minimum received total was around 5 dollars. This was just around the time of the hard fork and pre segwit. I'm curious to see how this turns out. Also, here is my personal email for sensitive detail and correspondences. I have a few inquiries that may require a bit more privacy. Thanks for the update and the best of luck👍🏻💯

Sent from Yahoo Mail on Android 
 
  On Mon, Feb 5, 2018 at 11:37 AM, Mitko J.<notifications@github.com> wrote:   
I'm gonna do new payment and let you know. Another question I've wanted to ask is there any minimal payment requirements or can be any amount ?

Once I've tested im gonna report with results.
